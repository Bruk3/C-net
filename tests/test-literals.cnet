10023
100.24
"Hello, World"
"Hello\nWorld"
"Hello'\r\t World"