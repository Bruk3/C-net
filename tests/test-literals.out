INTLIT(10023)
FLOATLIT(100.240000)
STRLIT(Hello, World)
STRLIT(Hello\nWorld)
STRLIT(Hello'\r\t World)
EOF